### PR TITLE
REST API: Merges changes from wpcom as of 09/04/2015

### PR DESCRIPTION
**Pull request by zinigor**

### CODE REVIEW

File: json-endpoints/class.wpcom-json-api-get-customcss.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	 */
+	function callback( $path = '', $blog_id = 0 ) {
+		// Switch to the given blog.
+		$blog_id = $this->api->switch_to_blog_and_validate_user( $this->api->get_blog_id( $blog_id ) );
+		if ( is_wp_error( $blog_id ) ) {
+			return $blog_id;
+		}
+
+		$args = array(
+			'css' => Jetpack_Custom_CSS::get_css(),
```
Comment (lezama) [inline json-endpoints/class.wpcom-json-api-get-customcss.php]: I think that the custom css module needs to be active in order to this class to be loaded. not sure :)

Comment (zinigor) [inline json-endpoints/class.wpcom-json-api-get-customcss.php]: Just for the record, we discussed that yesterday with @sirbrillig and he said that it's OK to leave this endpoint out for now. So I'm going to rebase this branch and remove this commit.

**Comment by zinigor** [comment]
I have tested this PR, everything works fine, should be good to merge after one final review.

**Comment by samhotchkiss** [comment]
Didn't test, but read through the changes, looks good to me

**Pull request merged**